libs/types-common/src/Data/Json/Util.hs

```diff
@@ -42,18 +49,35 @@ infixr 5 #
 
 -- | A newtype wrapper for 'UTCTime' that formats timestamps in JSON with
 -- millisecond precision instead of the default picosecond precision.
+-- Construct values using 'toUTCTimeMillis'; deconstruct with 'fromUTCTimeMillis'.
+-- Unlike with 'UTCTime', 'Show' renders ISO string.
+--
+-- It would be nice to have a Cql instance for this type to make storing slightly more
+-- straight-forward.  This would require cassandra-utils as a dependency here, as both brig and
+-- cassandra (and possibly others in the future) are using this type.
```

Comment:
You can surround Cql instances with 

```haskell
{-# LANGUAGE CPP                        #-}
#ifdef WITH_CQL
...
#endif
```
As is already done in other modules of types-common, e.g. `types-common/src/Data/Id.hs`

Cassandra, by default, stores timestamps in microseconds.